### PR TITLE
RAC-4019: Update config files for cit-fit conversions

### PR DESCRIPTION
**Pull request by hohene**
settings.py has been taught how to convert fit-based configuration items into the cit defaults config dictionary.

Many items that were previously found in cit_defaults.json have been removed as they now originate in the fit configuration files.
cit_defaults.json config still houses those items that don't have any fit equivalent.

@jimturnquist @johren @gpaulos @stuart-stanley

### CODE REVIEW

File: test/config/settings.py [hunk 1]
```diff
@@ -10,9 +10,36 @@
...
+        'RACKHD_CENTOS_REPO_PATH': '/CentOS/6.5',
+        'RACKHD_ESXI_REPO_PATH': '/ESXi/6.0',
+        'RACKHD_UBUNTU_REPO_PATH': '/Ubuntu/14'
+    }
+    for cit_path, local_path in mappings.items():
+        server_path = None
+        for proxy in fit_common.fitrackhd()['httpProxies']:
+            if local_path == proxy['localPath']:
+                server_path = proxy['server']
+                break;
```
Comment (jimturnquist) [inline test/config/settings.py]: I have addressed the above...

**Comment by hohene** [comment]
for my benefit:
working: http://rackhdci.lss.emc.com/job/RackHD/1517/
failed: http://rackhdci.lss.emc.com/job/RackHD/1518/

**Comment by hohene** [comment]
test this please

**Comment by PengTian0** [comment]
BUILD [RackHD #8](http://147.178.202.18//blue/organizations/jenkins/RackHD/detail/RackHD/8/pipeline) : FAILURE

**Comment by hohene** [comment]
The last build for the test passed: http://rackhdci.lss.emc.com/job/RackHD/1521/
@PengTian0 - The link you added above doesn't show me anything relevant, nor link to a build failing. Can you tell me what the issue is?

**Comment by johren** [comment]
test this please

**Pull request merged**